tgui/packages/tgui/interfaces/Pressurizer.js

```diff
@@ -0,0 +1,202 @@
+    act("fan", {
+      fanState,
+    });
+  };
+
+  const getArmedState = () => {
+    if (pressure<maxPressure*0.2)
+    { return "Insufficient Pressure"; }
+    if (!airSafe)
+    { return "AIR UNSAFE - Locked"; }
+    if (blastArmed)
+    { return "Armed"; }
+    else
+    { return "Ready"; }
+
+  };
+
+  const handleEjectContents = () => {
+    act("eject-materials");
+  };
+
+  const handleArmPressurizer = () => {
+    act("arm");
+  };
+
+  return (
+    <Window
+      theme={emagged ? 'syndicate' : 'ntos'}
+      width={390}
+      height={410}>
+      <Window.Content>
+        <Flex>
+          <Flex.Item width="900px">
+            <PortableBasicInfo
+              connected={connected}
+              pressure={pressure}
+              maxPressure={maxPressure}>
+              <Divider />
+              <LabeledList>
+                <LabeledList.Item label="Emergency Blast Release" >
+                  <Button
+                    fluid
+                    textAlign="center"
+                    icon="circle"
+                    content={getArmedState()}
+                    disabled={pressure<maxPressure*0.2 || !airSafe}
+                    color={blastArmed ? 'bad' : 'average'}
+                    onClick={() => handleArmPressurizer()} />
+                </LabeledList.Item>
+                <LabeledList.Item label="Delay">
+                  <Button
+                    onClick={() => handleSetBlastDelay(minArmDelay)}
+                    content="Min" />
+                  <NumberInput
+                    animated
+                    width="7em"
+                    value={blastDelay}
+                    minValue={minArmDelay}
+                    maxValue={maxArmDelay}
+                    onChange={(e, targetDelay) => handleSetBlastDelay(targetDelay)} />
+                  <Button
+                    onClick={() => handleSetBlastDelay(maxArmDelay)}
+                    content="Max" />
+                </LabeledList.Item>
+              </LabeledList>
+
+              <Divider />
+              <LabeledList>
+                <LabeledList.Item label="Fan Status">
+                  <Button
+                    content="Off"
+                    color={fanState === FanState.Off ? 'bad' : 'default'}
+                    onClick={() => handleSetFan(FanState.Off)} />
+                  <Button
+                    content="In"
+                    color={fanState === FanState.In ? 'good' : 'default'}
+                    onClick={() => handleSetFan(FanState.In)} />
+                  <Button
+                    content="Out"
+                    color={fanState === FanState.Out ? 'good' : 'default'}
+                    onClick={() => handleSetFan(FanState.Out)} />
+                </LabeledList.Item>
+                <LabeledList.Item label="Release pressure">
+                  <Button
+                    onClick={() => handleSetPressure(minRelease)}
+                    content="Min" />
```

Comment:
Can probably just put `Min` as a child rather than a `content` prop. More clear on what's happening.
```suggestion
                  <Button onClick={() => handleSetPressure(minRelease)}>
                    Min
                  </Button>
```
Same for other usages of `content="Foo"` in this file, where "Foo" is a single word or whatever.